pkg/cmd/projectCreate.go

```diff
@@ -660,6 +709,48 @@ func createTestFolderGradle(fileGenerator *utils.FileGenerator, targetTestFolder
 	return err
 }
 
+// Creates a build.gradle and a bnd.bnd file in a Gradle test project directory.
+func createManagerFolder(fileGenerator *utils.FileGenerator, targetTestFolderPath string,
```

Comment:
You are going to want to pass the manager name rather than the feature name here.